Meros.nimble

```diff
@@ -97,7 +97,7 @@ task unit, "Run unit tests.":
             .join(" ")
 
     #Ensure dependencies are installed.
-    nimbleExec "install --depsOnly"
+    nimbleExec "install --depsOnly -Y"
```

Comment:
Since Nimble uses a single location for all Nimble packages, this can be dangerous. Can't we have the automated scripts run `nimble install --depsOnly -Y` in advance so this `nimble install --depsOnly` doesn't trigger a prompt?